packages/compass-components/src/components/document-list/visible-field-toggle.tsx

```diff
@@ -58,32 +75,43 @@ const DocumentFieldsToggleGroup: React.FunctionComponent<{
     return null;
   }
 
+  const showButtonText = `Show ${showSizeDiff} more ${
+    showSizeDiff === 1 ? 'field' : 'fields'
+  }${parentFieldName ? ` in ${parentFieldName}` : ''}`;
+  const hideButtonText = `Hide ${hideSizeDiff} ${
+    hideSizeDiff === 1 ? 'field' : 'fields'
+  }${parentFieldName ? ` in ${parentFieldName}` : ''}`;
+
   return (
-    <div className={container}>
+    <div className={container} style={style}>
       {isShowButtonVisible && (
-        <Button
-          size="xsmall"
-          leftGlyph={<Icon glyph="ArrowDown"></Icon>}
+        <Link
+          as="button"
+          hideExternalIcon={true}
+          className={cx(linkButtonStyles, buttonClassName)}
           onClick={onShowClick}
-          className={button}
+          aria-label={showButtonText}
           data-testid="show-more-fields-button"
         >
-          Show {showSizeDiff} more fields
-        </Button>
+          <Icon size="xsmall" glyph="ArrowDown"></Icon>
+          {showButtonText}
+        </Link>
       )}
       {isHideButtonVisible && (
-        <Button
-          size="xsmall"
-          leftGlyph={<Icon glyph="ArrowUp"></Icon>}
+        <Link
+          as="button"
+          hideExternalIcon={true}
+          className={cx(linkButtonStyles, buttonClassName)}
           onClick={onHideClick}
-          className={button}
+          aria-label={hideButtonText}
           data-testid="hide-fields-button"
         >
-          Hide {hideSizeDiff} fields
-        </Button>
+          <Icon size="xsmall" glyph="ArrowUp"></Icon>
```

Comment:
I think in the designs that Ben shared it was CaretDown / CaretUp, not an arrow. Don't have a strong preference, but arrow seems a big big when compared to the text near it, so maybe worth trying